tests/feature_check/slice_attrs_check.py

```diff
@@ -0,0 +1,10 @@
+class A:
```

Comment:
Feature checks are for things that can't be simply skipped by printing SKIP in the file.  So in this case don't have a feature check script and just use the SKIP mechanism as you've done above.